addons/l10n_ar/models/l10n_latam_document_type.py

```diff
@@ -41,7 +41,7 @@ def _get_document_sequence_vals(self, journal):
         values.update({'padding': 8, 'implementation': 'no_gap', 'prefix': "%05i-" % (journal.l10n_ar_afip_pos_number),
                        'l10n_latam_journal_id': journal.id})
         if journal.l10n_ar_share_sequences:
-            values.update({'name': '%s - Letter %s Documents' % (journal.name, self.l10n_ar_letter),
+            values.update({'name': _('%s - Letter %s Documents' % (journal.name, self.l10n_ar_letter)),
```

Comment:
HI  @nicomacr 
Thank you for the PR :)

Here can you improve the way that the translation is made?
I mean remove the variables inside the _().
Maybe something similar like this?

Also please if you can add the translation term will be great, for localization modules we do it manually updating the pot and po file. 

```suggestion
            values.update({'name': _('%s - Letter %s Documents') % (journal.name, self.l10n_ar_letter),
```

Reply:
@zaoral Thanks for suggestions. I've already apply this in the current PR.